packages/devtools/CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 ## 0.0.19 - 2019-05-?? (in progress)
-* Update DevTools server to better handle failures when launching browsers
+* Update DevTools server to better handle failures when launching browsers.
+* Format URIs from user input.
```

Comment:
I don't think it's very clear from this what the change was (that we now accept more formats of VM service URIs).

Reply:
fixed.